src/device-registry/models/Airqloud.js

```diff
@@ -287,7 +287,11 @@ airqloudSchema.statics = {
   },
   async modify({ filter = {}, update = {} } = {}) {
     try {
-      let options = { new: true, useFindAndModify: false };
+      let options = {
+        new: true,
```

Comment:
just updating the projection to ensure that the response load is reduced, removed the HUGE location coordinates array from response after updating AirQloud